### DISCUSSION TITLE
Reverting to Rubberband for Pitch Shifting?

### DISCUSSION THREAD

**Discussion by KitsuneBalloon** [wwmm/easyeffects#3430] [Q&A] [answered]
I was wondering if i could revert back to Rubber Band Library for Pitch shifting since it worked so much better then SoundTorch for my system, i could've reverted back to 7.0.1 but i cannot downgrade back that far via flatpak, so i'm wondering if i can replace the SoundTorch Plugin with Rubberband again.

**Comment by wwmm** [comment, marked as answer]
These libraries have completely different parameters. So switching between them requires a lot of changes. 

In the past many users had problems with rubberband. For reasons we could never fully understand the system killed EasyEffects when rubberband was used. I speculate the reason was it being too heavy to be used in a realtime thread. But the thing is the problem does not happen with soundtouch. So we moved away from rubberband.

**Reply by KitsuneBalloon** [reply]
Fair enough, i personally never had any issues with Rubberband for pitch shifting, SoundTorch however doesn't sound very great, giving me segments of stuttering every 2 milliseconds, even on a decent CPU.

I suppose 7.0.1 is no longer available on flatpak?

Can't build it manually as it moans about tools that i already have, so i presume the tools are just not the version it's looking for.
